docs/source/migrations/modes.rst

```diff
@@ -0,0 +1,46 @@
+======================
+Snuba Migration Modes
+======================
+
+The main "switch" between the two modes for running data migrations (local and
+distributed) lives in a user's local Sentry configuration (in their ``sentry.conf.py``).
+The controlling envrionment variable is ``SENTRY_DISTRIBUTED_CLICKHOUSE_TABLES``,
+and its default value is set `at this line <https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L127>`_.
```

Comment:
I would not add a link to the server.py file if not strictly necessary. 
That file change, the link would not so it will eventually point to the wrong place.
Giving the name of the setting is enough here.

Reply:
Good point; I removed the link.